spec/coffee-script-spec.coffee

```diff
@@ -462,6 +462,22 @@ describe "CoffeeScript grammar", ->
       expect(tokens[9]).toEqual value: 'c', scopes: ['source.coffee', 'string.quoted.double.coffee']
       expect(tokens[10]).toEqual value: '"', scopes: ['source.coffee', 'string.quoted.double.coffee', 'punctuation.definition.string.end.coffee']
 
+      {tokens} = grammar.tokenizeLine('"\\\\a\\\\t\\\\a\\\\b"')
```

Comment:
I think there's a slight misunderstanding - what I'm asking for is something like `'"\\a\\t\\a\\b"'`.